#! /bin/sh /usr/share/dpatch/dpatch-run
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: 40_scripts__mysql_install_db.sh__no_test.patch by  <ch@debian.org>

@DPATCH@

40_scripts__mysql_install_db.sh__no_test.patch by  <ch@debian.org>

scripts__mysql_install_db.sh__no_test
http://bugs.mysql.com/bug.php?id=6901

--- a/scripts/mysql_install_db.sh
+++ b/scripts/mysql_install_db.sh
@@ -351,7 +351,7 @@
 fi
 
 # Create database directories
-for dir in $ldata $ldata/mysql $ldata/test
+for dir in $ldata $ldata/mysql
 do
   if test ! -d $dir
   then
